core/common/src/main/java/alluxio/metrics/Metric.java

```diff
@@ -300,32 +319,31 @@ public static String getMetricNameWithUserTag(String metricName, String userName
   public static Metric from(String fullName, double value, MetricType metricType) {
     String[] pieces = fullName.split("\\.");
     Preconditions.checkArgument(pieces.length > 1, "Incorrect metrics name: %s.", fullName);
-
+    int len = pieces.length;
```

Comment:
Fix a bug that the new metrics name is `<Instance_type>.<MetricName>.Tags.<InstanceHostname>` instead of `<Instance_type>.<MetricName>.<InstanceHostname>.Tags`